src/firewall/core/io/policy.py

```diff
@@ -570,6 +570,16 @@ def common_check_config(obj, config, item, all_config, all_io_objects):
                             log.debug1("{} (unsupported)".format(ex))
                         else:
                             raise ex
+            elif isinstance(obj_rich.element, rich.Rich_Service):
+                if "services" in all_io_objects:
```

Comment:
nit: The `if` guard is not necessary. The `all_io_objects` dict will always have a "services" key. The `existing_services` temporary variable is also unnecessary, but okay if you think it improves readability.

Reply:
I was thinking so too and wanted to avoid the check, but the block above has a similar check.

If I can rely on that, all the better. Will change...